iis-workflows/iis-workflows-transformers/src/main/resources/eu/dnetlib/iis/workflows/transformers/referenceextraction/project/filter/oozie_app/lib/scripts/transformer.pig

```diff
@@ -0,0 +1,14 @@
+define avro_load_project
+org.apache.pig.piggybank.storage.avro.AvroStorage(
+'schema', '$schema');
+
+define avro_store_project
+org.apache.pig.piggybank.storage.avro.AvroStorage(
+'index', '0',
+'schema', '$schema');
+
+project = load '$input' using avro_load_project;
+
+outputProject = FILTER project BY (fundingClass is null) or (NOT(fundingClass MATCHES '$fundingclass_blacklist_regex'));
```

Comment:
It's funny how implementing this functionality was so easy in Pig while the whole wrapping of this code is huge.